agent-service/tests/test_rag_smoke.py

```
@@ -0,0 +1,61 @@
+            logger=logging.getLogger("test"),
+        )
+    )
+
+    assert result["content"] == "Stub answer"
+    assert result["sources"] == []
+    assert result.get("note") is None
+
+
+def test_llm_unavailable_returns_placeholder(monkeypatch):
+    def _llm_unavailable():
+        raise RuntimeError("LLM unavailable")
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)